interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md

```diff
@@ -15,25 +15,52 @@ To add the **Picture Box** control to a report, drag the **Picture Box** item fr
 
 Specify one of the following properties to set an image:
 
-- **ImageSource**  
-    Use this property to save images along with a report definition.
-- **ImageUrl**  
-    Use this property to save only the path to the image.
+- **Image Source**  
+    Save the image to the report's definition.
+- **Image Url**  
+    Save only a path to the image.
 
 ![](../../../../../images/eurd-win-picture-box-image-property.png)
 
+## Assign an External Image
+
+Click the **Image Source** / **Image URL** property's ellipsis button and the **Open File** dialog is invoked.
+
+![](../../../../../images/eurd-win-ImageSource-OpenFileDialog.png)
+
+The selected image / its URL is saved to the report definition **.repx** file.
+
+## Assign an Image from the Report's Image Collection
+
+Set the report's **Image Resources** property.
+
+![](../../../../../images/eurd-win-ImageResources-Editor.png)
+
+Invoke the **Picture Box**'s smart tag. Click the **Expression** option's ellipsis button to invoke the **Expression Editor**. Choose an image from the **Images** collection:
+
+![ImageSource-OpenFileDialog](../../../../../images/eurd-win-ImageSource-ExpressionEditor-ImagesCollection.png)
+
+
 ## Bind a Picture Box to Data
-You can use the Picture Box to display an image [dynamically obtained](../../bind-to-data/bind-controls-to-data-expression-bindings.md) from a data source. Click the control's smart tag, expand the **Image** property's **Expression** drop-down list and select the data field.
 
-![](../../../../../images/eurd-win-picture-box-bind-to-data.png)
+Use one of the following techniques to add the **Picture Box** control that obtains an image from a data source.
```

Comment:
Change "the **Picture Box** control that obtains an image from a data source" to "the **Picture Box** control, which obtains an image from a data source"